src/itest/kotlin/nl/lifely/zac/itest/client/ZacClient.kt

```diff
@@ -168,8 +168,8 @@ fun createZaakAfhandelParameters() {
             "        \"type\": \"HUMAN_TASK\"\n" +
             "      }\n" +
             "    ],\n" +
-            "    \"key\": \"melding-klein-evenement\",\n" +
-            "    \"naam\": \"Melding klein evenement\",\n" +
+            "    \"key\": \"generiek-zaakafhandelmodel\",\n" +
```

Comment:
Ideally at least the key (maybe also the name) would be in English but yeah.. not sure how to translate this easily.. I guess let's leave this for now unless you have a good idea? ;-)